sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/CommunicationIdentifierSerializer.java

```diff
@@ -1,10 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.android.communication.common;
+package com.azure.android.communication.chat;
 
 import android.text.TextUtils;
 
+import com.azure.android.communication.chat.models.CommunicationCloudEnvironmentModel;
+import com.azure.android.communication.chat.models.CommunicationIdentifierModel;
+import com.azure.android.communication.chat.models.CommunicationUserIdentifierModel;
+import com.azure.android.communication.chat.models.MicrosoftTeamsUserIdentifierModel;
+import com.azure.android.communication.chat.models.PhoneNumberIdentifierModel;
+import com.azure.android.communication.common.CommunicationCloudEnvironment;
+import com.azure.android.communication.common.CommunicationIdentifier;
+import com.azure.android.communication.common.CommunicationUserIdentifier;
+import com.azure.android.communication.common.MicrosoftTeamsUserIdentifier;
+import com.azure.android.communication.common.PhoneNumberIdentifier;
+import com.azure.android.communication.common.UnknownIdentifier;
```

Comment:
Are these imports all required by a one-liner code change? I doubt

Reply:
I checked code they are used, maybe because the file is now moved to chat package and need explicit import.